packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -3,12 +3,11 @@
   <id>app.organicmaps.desktop</id>
 
   <name>Organic Maps</name>
-  <summary>A free offline maps app for travelers, tourists, hikers, and cyclists based on top of crowd-sourced OpenStreetMap data and curated with love by MapsWithMe (Maps.Me) founders</summary>
```

Comment:
Let's leave this line as is, for better SEO.

Reply:
I don't agree. This is a point of this PR. The summary [is too long according to wise quality guidelines](https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines/#not-too-long-1).
Take a look at:

![image](https://github.com/organicmaps/organicmaps/assets/14926724/f8e698e7-6106-4218-8f33-45fcd4b1cf41)

Reply:
I see, thanks. Let's cut it off then, but add `,drivers ` before `and cyclists`.